anthropic/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "53fe7c3e-57eb-42ca-8e43-ec92c04b6160",
+  "app_id": "anthropic",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Optimize Anthropic usage: cost estimates, prompt sampling and performance metrics.",
+    "title": "Anthropic",
+    "media": [],
+    "classifier_tags": [
+      "Category::AI/ML",
+      "Category::Metrics",
+      "Category::Cost Management",
+      "Submitted Data Type::Traces",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
+      "source_type_id": 31102434,
+      "source_type_name": "Anthropic",
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "anthropic.",
+        "check": [],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
```

Comment:
@dpoint01 Do we have any dashboards we would like to add here

Reply:
It should just be LLM Observability Overview Dashboard